dataset/dictionary/enemies.json5

```diff
@@ -2179,4 +2179,24 @@
     tags: [ "fontaine", "enemy" ],
     notes: "フォンテーヌの地方伝説の1体で、プクプク獣の変異体",
   },
+  {
+    en: "Hydro Tulpa",
+    ja: "水形タルパ",
+    zhCN: "水形幻人",
+    tags: [ "fontaine", "enemy" ],
+  },
+  {
+    en: "Maintenance Mek: Gold Leader",
+    ja: "メンテマシナリー·黄金統率型",
+    zhCN: "维护机关·澄金领队型",
+    pronunciationJa: "メンテマシナリーおうごんとうそつがた",
+    tags: [ "fontaine", "enemy" ],
+  },
+  {
+    en: "All-Devouring Narwhal",
+    ja: "呑星の鯨",
+    zhCN: "吞星之鲸",
+    pronunciationJa: "どんせいのくじら",
+    tags: [ "fontaine", "enemy" ],
```

Comment:
@SleepyAsh0191 Sorry for the review comment after merge.
Isn't this a weekly boss? (Actually, I haven't played the 4.2 Archon Quest yet, so I might be wrong.)
If All-Devouring Narwhal is a weekly boss, `enemy-boss` is the better categorization than `enemy`.

Reply:
sorry mischeck it's enemy-boss

Reply:
> @SleepyAsh0191 Sorry for the review comment after merge. Isn't this a weekly boss? (Actually, I haven't played the 4.2 Archon Quest yet, so I might be wrong.) If All-Devouring Narwhal is a weekly boss, `enemy-boss` is the better categorization than `enemy`.

i will fix it in next contribute

Reply:
No problem, thanks! 😊